src/toolbox.js

```diff
@@ -205,10 +221,13 @@ var Toolbox = GObject.registerClass({
         }
 
         const topic = row.get_child();
-        this._headerLabel.label = topic.title;
-        this._headerImage.iconName = topic.icon_name;
-        this._topicsStack.visibleChildName = topic.id;
-        this._setMinimized(false);
+        if (topic.sensitive) {
+            this._headerLabel.label = topic.title;
+            this._headerImage.iconName = topic.icon_name;
+            this._topicsStack.visibleChildName = topic.id;
+            this._setMinimized(false);
+            topic.emit('clicked');
```

Comment:
My understanding was that we should 1) not show the topic panel when the insensitive button is clicked; and 2) when we have a topic panel that is already shown (for example by default or because the topic was previously selected by the user) then the child widgets should all be disabled

Reply:
I see your point though, and that isn't a case I thought of during my meeting with Robin. I guess we can either implement it the way you mentioned or we could also disable the panel collapse button. I'll check with Robin.

Reply:
Yes, I can confirm, Ben designed the quest to display messages when the player clicks on insensitive topics. At least from the quest, the clicked signal is only relevant for disabled topics.

```
        # if the user flips the app and clicks on a coding panel, Faber responds as follows:
        # LIGHTSPEEDKEY_PANEL_LOCKED - if the panel they've clicked on is not available this level
        # LIGHTSPEEDKEY_PANEL_GAME- if they click on the Game panel
        # LIGHTSPEEDKEY_PANEL_SPAWN - if they click on the Spawn panel
        # LIGHTSPEEDKEY_PANEL_ASTEROID - if they click on the Asteroid panel
        # LIGHTSPEEDKEY_PANEL_SPINNER - if they click on the Spinner panel
        # LIGHTSPEEDKEY_PANEL_SQUID - if they click on the Squid panel
        # LIGHTSPEEDKEY_PANEL_BEAM - if they click on the Beam panel
        # LIGHTSPEEDKEY_PANEL_POWERUPS - if they click on the PowerUps panel
```

Reply:
Sorry, there are messages for clicking in insensitive and sensitive topics. Like:
> This panel isn't available on this level!  How can you solve it with another panel?
> The code in these panels creates behaviors of an enemy type. This panel works for Squids

Reply:
Thanks for the clarification!